stl_text/ops/vocab/pytext_vocab.py

```diff
@@ -1,28 +1,59 @@
 #!/usr/bin/env python3
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
-from typing import List, Optional
-
+import torch
 import torch.nn as nn
-from pytext.data.bert_tensorizer import build_fairseq_vocab
 from pytext.data.utils import BOS, EOS, MASK, PAD, UNK
-from pytext.torchscript.vocab import ScriptVocabulary
-from pytext.utils.file_io import PathManager
+from typing import Dict, List, Optional
+from iopath.common.file_io import PathManager
+from torchtext.experimental.vocab import (
+    load_vocab_from_file,
+    build_vocab_from_iterator,
+)
 
-SPECIAL_TOKEN_REPLACEMENT = {
-    "[UNK]": UNK,
-    "[PAD]": PAD,
-    "[CLS]": BOS,
-    "[MASK]": MASK,
-    "[SEP]": EOS,
-}
+class VocabTransform(nn.Module):
+    r""" Wrapper around torchtext.experimental.vocab.Vocab 
 
+    Args:
+        vocab_path (Optional[str]): Path to vocabulary file. The file should contain tokens seperated by new line.
+        vocab_list (Optional[List[str]]): List of vocabulary tokens
+        add_bos (bool): Whether to add BOS token in the beginning of sentence
+        add_eos (bool): whether to add EOS token at the end of sentnce
+        max_seq_len (int): The maximum length of sequence. The sequence will be truncated max_seq_len if input length is larger than this value.
 
-class VocabTransform(nn.Module):
+    Example:
+        >>> from stl_text.ops.vocab import VocabTransform
+        >>> vocab = VocabTransform("vocab.txt")
+        >>> jit_vocab = torch.jit.script(vocab)
+        >>> print(jit_vocab.code)
```

Comment:
What is the reasoning behind printing the code here as well?

Reply:
> What is the reasoning behind printing the code here as well?

Thanks for pointing this out. It was just part of Bootcamp task, but we really don't need to keep it. I will remove it from the final version.